projects/plugins/jetpack/extensions/plugins/publicize/index.js

```diff
@@ -9,19 +9,22 @@
  */
 
 import { JetpackLogo } from '@automattic/jetpack-components';
-import { TwitterThreadListener } from '@automattic/jetpack-publicize-components';
+import {
+	TwitterThreadListener,
+	useSocialMediaConnections as useSelectSocialMediaConnections,
```

Comment:
Just out of curiosity, why did you rename this hook?

Reply:
copypasta

Reply:
I'll remove

Reply:
Addressed in 19baa835576d2c002ec9d97ce0909517c6ba082b